dcos/dcos-ui
# DCOS-57614: table-multiselect doesn't work as expected

Allow multiple job run to be selected
in the job run history table

Closes https://jira.mesosphere.com/browse/DCOS-57614

## Testing
1. Go to jobs tab.
2. Create a job that wouldn't finish immediately.
Example JSON:
```
{
  "id": "new-job-1",
  "description": "",
  "run": {
    "cpus": 1,
    "disk": 0,
    "mem": 128,
    "ucr": {
      "image": {
        "id": "nginx",
        "kind": "docker"
      }
    },
    "cmd": "sleep 1000"
  }
}
```
3. Open it.
4. Run it at least once.
5. In the runs history table, verify that the actions dropdown exists and it contains the stop actions. It should only exist for running instances.
6. Verify that clicking stop works.
7. Verify that overall the table works the same way as before.
8. Verify that the added test makes sense.

## Trade-offs
~https://github.com/dcos/dcos-ui/pull/4069#discussion_r314611962~

## Dependencies
None.

## Screenshots
### Before
![Peek 2019-08-15 12-46-2](https://user-images.githubusercontent.com/40791275/63087420-d4395600-bf5a-11e9-9806-bd1ad7d04d7e.gif)

### After
![Peek 2019-08-23 15-43](https://user-images.githubusercontent.com/40791275/63594226-97a2d580-c5be-11e9-9f82-831cb16eb4d9.gif)
